qemu/tests/snp_basic_config.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+from avocado.utils import process
+from virttest import data_dir as virttest_data_dir
+from virttest import error_context
+from virttest.utils_misc import verify_dmesg
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu snp basic test on Milan and above host:
+    1. Check host snp capability
+    2. Boot snp VM
+    3. Verify snp enabled in guest
+    4. Check snp qmp cmd and policy
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    error_context.context("Start sev-snp test", test.log.info)
+    timeout = params.get_numeric("login_timeout", 240)
+
+    snp_module_path = params["snp_module_path"]
+    if os.path.exists(snp_module_path):
+        with open(snp_module_path) as f:
+            output = f.read().strip()
+        if output not in params.objects("module_status"):
+            test.cancel("Host sev-snp support check fail.")
+    else:
+        test.cancel("Host sev-snp support check fail.")
+    socket_count_cmd = params.get("socket_count_cmd")
+    if socket_count_cmd:
+        if int(process.getoutput(socket_count_cmd, shell=True)) != 1:
+            test.cancel("Host cpu has more than 1 socket, skip the case.")
+
+    vm_name = params["main_vm"]
```

Comment:
I think we need to add env_process.preprocess_vm(test, params, env, vm_name) after line 39

Reply:
Thanks Xueqiang, this is discussed with zhenchao, since I don't change configs during the test, the preprocess can be removed since VT will do it by default at the beginning of the test.

Reply:
ok, make sense.